pyqt-apps/siriushla/as_ap_currinfo/charge_monitor.py

```diff
@@ -51,9 +51,7 @@ def _setupUi(self):
         self.timeplot.setStyleSheet(
             '#timeplot{min-width:28em; min-height: 18em;}')
         t_end = Time(datetime=_datetime.now())
```

Comment:
if we derive the class `Time` from `datetime`, I think we could create an `staticmethod` called now that would return a `Time` object, not a `datetime` object. It seems useful!

Reply:
done! 
now, the use of the Time class seems to be cleaner and more intuitive!